src/components/changePlanner/CPDetailView.tsx

```diff
@@ -358,157 +392,196 @@ class CPDetailView extends React.Component<
     }
     return (
       <div className={Classes.DARK + " asset-view"}>
+        <Alert
+          cancelButtonText="Cancel"
+          confirmButtonText="Execute"
+          intent={Intent.WARNING}
+          className={Classes.DARK}
+          isOpen={this.state.isAlertOpen}
+          onCancel={() => this.handleExecuteCancel()}
+          onConfirm={() => this.handleExecute()}
+        >
+          <p>Are you sure you want to execute this change plan? You will not be able to generate a work order for this change plan anymore</p>
+        </Alert>
         <Toaster
           autoFocus={false}
           canEscapeKeyClear={true}
           position={Position.TOP}
           ref={this.refHandlers.toaster}
         />
-        <h1>Change Plan</h1>
-        <div className="detail-buttons-wrapper">
-          <div className={"detail-buttons"}>
-            <AnchorButton
-              minimal
-              intent="primary"
-              icon="document-open"
-              text="Generate Work Order"
-            />
-          </div>
-          {this.loading ? <Spinner /> : null}
+        <h1>
+          Change Plan:
```

Comment:
also no space after `Plan:` in the view

Reply:
i swear i have this fixed on my next PR !!